tensorboard/plugins/profile/input_pipeline_analyzer/input-pipeline-analyzer.html

```diff
@@ -142,28 +142,33 @@
         <div id="title_host_side_analysis">
           <p class="section-header">Section 3: Host-side analysis details</p>
         </div>
-        <div style="height:300px;" id="host_side_chart"></div>
-        <p>
-          <span><b>Explanation:</b> </span>
-          <span>To improve performance, you should try to convert from </span>
-          <span>"Reading data from files on demand" (if any) to </span>
-          <span>"Reading data from files in advance". </span>
-        </p>
+        <div hidden$="[[!_show_host_side_chart]]">
+          <div style="height:300px;" id="host_side_chart"></div>
+          <p>
+            <span><b>Explanation:</b> </span>
+            <span>To improve performance, you should try to convert from </span>
+            <span>"Reading data from files on demand" (if any) to </span>
+            <span>"Reading data from files in advance". </span>
+          </p>
+        </div>
+        <div hidden$="[[_show_host_side_chart]]">
+          Host side analysis can not be done without instrumentation.
+        </div>
         <p> Click the "Show" button below to see the source data of the breakdown.</p>
         <button on-click="onClick">[[_toggle_button_text]]</button>
-        <template is="dom-if" if="{{_show_host_side_table}}">
-          <table id="host-side-table", style="width:100%;">
-            <thead>
-              <th>Input Op</th>
-              <th>Count</th>
-              <th>Total Time (in ms)</th>
-              <th>Total Time (as % of total input-processing time)</th>
-              <th>Total Self Time (in ms)</th>
-              <th>Total Self Time (as % of total input-processing time)</th>
-              <th>Category</th>
-            </thead>
-          </table>
-        </template>
+        <table style="width:100%;" hidden$="[[!_show_host_side_table]]">
```

Comment:
Could we add a class name to this table, ie `class="performance-table"` and then in the CSS, specify

```css
.performance-table {
  width: 100%;
}
```

? That separates content from styling.

Reply:
done